src/app/i18n/index.js

```diff
@@ -0,0 +1,21 @@
+import { createInstance } from 'i18next'
```

Comment:
I think this folder would be better put inside `/lib` instead of the app router folder. I think it would create a route for this?